chi2/chinotype.py

```diff
@@ -98,7 +98,11 @@ def __init__(self, listargs=[], args={}):
         self.crc_user = db['crc_user']
         self.crc_service =  db['crc_service_name']
         self.crc_pw = db['crc_pw']
+        self.branchnodes = db['chi_branchnodes']
+        self.allbranchnodes = db['chi_allbranchnodes']
```

Comment:
Config variable for further limiting what branch nodes are chosen (Currently ones with an 'F%' visual attribute, i.e. folders, and more than 10 patients below them. Might actually want to raise that bar.)